### PR TITLE
Blog&news

### PR DESCRIPTION
**Pull request by JRybelis**
- Layout of section elements in HTML.
- Responsive design styling.
- Static CSS styling.
- .hover-link class styling in home-page.css

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -461,7 +461,66 @@ <h6 class=author-name>Steven King</h6>
     <!-- section BLOG & NEWS begin -->
     <section class="container">
         <div class="row">
-            BLOG & NEWS
+            <div class="col-4 col-md-6 col-sm-8 col-xs-12">
+                <h6>blogs & news</h6>
+                <h3> Interesting articles <span class="news-frequency">updated daily</span> <!-- contains a section of text to be styled separately from the rest of the text block--> </h3>
```
Comment (pigma15) [inline index.html:466]: ```<span>``` can be changed to ```<u> ```so it will follow rules from ```font.css```

File: css/components/blog-news.css [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    opacity: 0;
+    transform: translateX(-100%);
+}
+.news-column> li > a::after {
+   
+}
+.news-frequency {
+    font-size: 32px;
+    color: var(--font-blue);
+    font-weight: 700;
```
Comment (pigma15) [inline css/components/blog-news.css:34]: font weight does not work with these font styles

**Review by pigma15 — APPROVED** [review]
Of course, changes are going to be needed as Jokubas stated in discord, but for now it lgtm.

**Review by bemze — APPROVED** [review]
Prisidedu prie Andriaus. Situacija zinoma :)

**Pull request merged**